### DISCUSSION TITLE
Configure the amount of digits shown after the decimal point

### DISCUSSION THREAD

**Discussion by neodd70** [VasilKalchev/LiquidMenu#71] [Ideas]
First off, awesome library.....I've been looking for a menu library for a long time that I can understand with my limit coding skills and yours is very understandable. Now to my issue. I am trying to display a float variable on a 20x4 LCD display with 3 places past the decimal. I was able to do this when I was just using the lcd.print by passing in the decimal place argument after the variable. I am not seeing where I can do this in the LiquidLine setup and everything I try ends up with a compile error. Again my coding skills are very basic and this may be something that can be done outside of the LiquidMenu scope but I'm not sure how. Any help would be greatly appreciated.

**Comment by VasilKalchev** [comment]
Thank you and sorry for the delay. There is a function now that allows you to set the number of decimal places per LiquidLine object: `void LiquidLine::set_decimalPlaces(uint8_t count);`. The function is only available in the working repository (it won't update through Arduino's library manager for now), if you want to use it you need to download it from [here](https://github.com/VaSe7u/LiquidMenu/archive/master.zip).
